semantic-reranker/src/main/java/org/opensearch/search/relevance/model/dto/Document.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.search.relevance.model.dto;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+
+import java.util.List;
+
+@JsonNaming(PropertyNamingStrategies.UpperCamelCaseStrategy.class)
+public class Document {
+  
+  private String id;
+  private String title;
```

Comment:
The existence of the `title` field looks like it's just part of the Kendra reranking API, and this plugin always passes `null`. 

The only fields that matter for this plugin are `id`, `tokenizedBodyPassages`, and `originalScore`, right?

(I'm also assuming that these classes will be replaced with a proper SDK client model.)

Reply:
We are still figuring out the user experience for configuring a mapping between fields in their document and the `title` and `body` expected by the ranker, so I will address this in a follow up.